client/src/api/attendees.js

```diff
@@ -0,0 +1,21 @@
+function saveAttendee(attendee) {
+  return fetch(`/api/attendees`, {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json"
+    },
+    body: JSON.stringify({
+      name: attendee.name,
```

Comment:
Isn't `JSON. stringify(attendee) ` enough?

Reply:
I'm not sure.
I tried it with JSON Server like this

```
async function handleSubmit(event) {
    event.preventDefault();
    await fetch("http://localhost:8080/attendees", {
      method: "POST",
      headers: {
        "Content-Type": "application/json"
      },
      body: JSON.stringify({
        state
     })
    });
  }
```
and got this in the db.json

```
"attendees": [
    {
"state": {
        "name": "Max Muster",
        "address": "Musterweg 1",
        "location": "",
        "email": "",
        "phone": "",
        "workshop1": "",
        "workshop2": "",
        "workshop3": "",
        "else": ""
      }
...
}
]
```
Alex and me thought, that could be a problem later.
Or should have it work with `body: JSON.stringify(state)` ?